staff/carlos-diestro/pre-course/bingo/bingo.js

```diff
@@ -0,0 +1,90 @@
+var card = [];
+var numbers = [];
+var matches = [0, 0, 0];
+
+bingo();
+
+function bingo() {
+  fillCard(3, 5);
+
+  numbers = [];
+
+  console.log(card);
+
+  newTurn(card, matches);
+
+  console.log("Has cantado bingo en " + numbers.length + " turnos");
+}
+
+function fillCard(rows, columns) {
+  for(var i = 0; i < rows; i++) {
+    var row = [];
+
+    for(var j = 0; j < columns; j++) {
+      row.push(getUniqueNumber());
+    }
+
+    card.push(row);
+  }
+}
+
+function getNumber() {
+  return Math.floor(Math.random() * (91 - 1) + 1);
+}
+
+function getUniqueNumber() {
+  var number = getNumber();
+
+  while(numbers.indexOf(number) > -1) {
+    number = getNumber();
+  }
+
+  numbers.push(number);
+
+  return number;
+}
+
+function newTurn(card) {
+  var play = confirm("¿Quieres seguir jugando?");
+
+  if(play) {
+    var newCard = JSON.parse(JSON.stringify(card));
+    var number = getUniqueNumber();
+
+    console.log("Ha salido el numero " + number);
+
+    var allMatches = checkNumber(newCard, number);
+
+    console.log(newCard);
+
+    if(allMatches == 15) {
```

Comment:
I think it is a better practice to use always === for comparisons

Reply:
It depends on the situation.

The strict mode is used when you are expecting same value and type but in my case I don't care about the type.

Anyways I appreciate your suggestion, thanks!